src/components/map/RouteMap.scss

```
@@ -0,0 +1,22 @@
+.map {
+  position: relative;
+
+  img {
+    padding: 5%;
+    width:90%;
```

Prefer spaces after colons
```suggestion
    width: 90%;
```